assignment1/KthToLastElement_Test.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+from linked_list import LinkedList, kth_to_last_element
+import unittest
+
+class KthToLastElement_Test(unittest.TestCase):
+
+    def setUp(self):
+        self.__linked_list = LinkedList()
+        self.__linked_list.add_first(0)
+        self.__linked_list.add_first(1)
+        self.__linked_list.add_first(2)
+        self.__linked_list.add_first(3)
+        self.__linked_list.add_first(4)
+        self.__linked_list.add_first(5)
+        self.__linked_list.add_first(6)
+        self.__linked_list.add_first(7)
+        self.__linked_list.add_first(8)
+        self.__list_head = self.__linked_list.get_head()
+        
+    def testBaseCases(self):
+        self.__max_value = self.__linked_list.size()
```

Comment:
You should either initialize `self.__max_value` in `setUp()` or you should use a local variable instead.
Note that, at the moment, if you run only the second test `testErrorCases` without running this test, the variable `self.__max_value` will not be initialized.